From 2c06a193b3d6e86a91230f18af02d1bcfcbb23d1 Mon Sep 17 00:00:00 2001
From: Petr Oros <poros@redhat.com>
Date: Wed, 17 Sep 2014 16:24:22 -0400
Subject: [redhat] kabi: Adding symbol pagevec_lookup_tag (mm/swap.c)

Message-id: <7f84c7164ab124579b0fa7e5a510a0ced45b08e9.1410510773.git.poros@redhat.com>
Patchwork-id: 96021
O-Subject: [PATCH RHEL7.1 142/233] kabi: Adding symbol pagevec_lookup_tag (mm/swap.c)
Bugzilla: 1071098
RH-Acked-by: Larry Woodman <lwoodman@redhat.com>

Bugzilla: 1071098
    https://bugzilla.redhat.com/show_bug.cgi?id=1071098

    Description:
    ============
    Lookup pages in the pagecache with specified tag, usually DIRTY tag.

Signed-off-by: Petr Oros <poros@redhat.com>
---
 create mode 100644 redhat/kabi/kabi-module/kabi_ppc64/pagevec_lookup_tag
 create mode 100644 redhat/kabi/kabi-module/kabi_s390x/pagevec_lookup_tag
 create mode 100644 redhat/kabi/kabi-module/kabi_x86_64/pagevec_lookup_tag

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/redhat/kabi/kabi-module/kabi_ppc64/pagevec_lookup_tag b/redhat/kabi/kabi-module/kabi_ppc64/pagevec_lookup_tag
new file mode 100644
index 0000000..22755da
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_ppc64/pagevec_lookup_tag
@@ -0,0 +1,2 @@
+#1-
+0x35d5d8c7 pagevec_lookup_tag vmlinux EXPORT_SYMBOL
diff --git a/redhat/kabi/kabi-module/kabi_s390x/pagevec_lookup_tag b/redhat/kabi/kabi-module/kabi_s390x/pagevec_lookup_tag
new file mode 100644
index 0000000..a5c99fb
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_s390x/pagevec_lookup_tag
@@ -0,0 +1,2 @@
+#1-
+0x9244e903 pagevec_lookup_tag vmlinux EXPORT_SYMBOL
diff --git a/redhat/kabi/kabi-module/kabi_x86_64/pagevec_lookup_tag b/redhat/kabi/kabi-module/kabi_x86_64/pagevec_lookup_tag
new file mode 100644
index 0000000..fc1a46e
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_x86_64/pagevec_lookup_tag
@@ -0,0 +1,2 @@
+#1-
+0xba1665aa pagevec_lookup_tag vmlinux EXPORT_SYMBOL
-- 
1.7.1